api/v1beta2/ibmvpccluster_types.go

```diff
@@ -90,13 +90,14 @@ type VPCLoadBalancerStatus struct {
 	// id of VPC load balancer.
 	// +optional
 	ID *string `json:"id,omitempty"`
-	// +optional
-	Name *string `json:"name,omitempty"`
```

Comment:
why are we removing the name?

Reply:
Initially I thought of having [Loadbalaners](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/api/v1beta2/ibmpowervscluster_types.go#L134) in status as slice so name was required to identify a loadbalancer, During our api review we decided to have map instead of list hence the name is no more required.